frontend/src/lib/components/HistoryList/HistoryList.scss

```diff
@@ -0,0 +1,21 @@
+@import '~/vars.scss';
+
+.history-list {
+    .history-list-row {
+        display: flex;
+        margin-top: $default_spacing;
```

Comment:
`$default_spacing` doesn't seem valuable, I'm a proponent of just using `1rem` directly, for clarity

Reply:
When I was new to the code `$default_spacing` was useful to help discover and navigate. So, at least in some contexts, having a name for the thing is clearer.

If we're going to stop using it we should probably have a separate PR that removes it so we can discuss the impact... 

I'd wonder two things

1) how does someone knew to the code know to use 1rem for spacing without looking at lots of files
2) how do we stop the value we use for spacing from being slightly different everywhere

Reply:
Since I don't know any better, I wouldn't block the PR for this. 🤷 

@Twixes time to make a mental note that we need to come up with a short document, or dare I say guidelines, that spells out all the need to knows about our design system.